middleware/auth.js

```
@@ -0,0 +1,31 @@
+  req.jwt = jwt;
+
+  if (!jwt) {
+    return res.status(401).json({
+      success: false,
+      message: "JWT Authentication Required (Try Logging In)",
+    });
+  }
+
+  jwt = jwtP.decode(jwt);
+  let jwtId;
+  if (jwt && jwt.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)